server/app/database/data/items.json

```diff
@@ -132514,5 +132639,1109 @@
+      "it": "[!] Torque de Ravageur",
+      "pt": "[!] Torque de Ravageur"
+    },
+    "itemType": "Amulet",
+    "setID": "498",
+    "level": 200,
+    "stats": [
+      {
+        "stat": "Vitality",
+        "minStat": 351,
+        "maxStat": 400
+      },
+      {
+        "stat": "Chance",
+        "minStat": 41,
+        "maxStat": 50
+      },
+      {
+        "stat": "Agility",
+        "minStat": 41,
+        "maxStat": 50
+      },
+      {
+        "stat": "Wisdom",
+        "minStat": 21,
+        "maxStat": 30
+      },
+      {
+        "stat": "Critical",
+        "minStat": 2,
+        "maxStat": 3
+      },
+      {
+        "stat": "AP",
+        "minStat": null,
+        "maxStat": 1
+      },
+      {
+        "stat": "Range",
+        "minStat": null,
+        "maxStat": 1
+      },
+      {
+        "stat": "Water Damage",
+        "minStat": 9,
+        "maxStat": 12
+      },
+      {
+        "stat": "Air Damage",
+        "minStat": 9,
+        "maxStat": 12
+      },
+      {
+        "stat": "Dodge",
+        "minStat": 4,
+        "maxStat": 5
+      },
+      {
+        "stat": "Pushback Damage",
+        "minStat": 21,
+        "maxStat": 30
+      },
+      {
+        "stat": "% Earth Resistance",
+        "minStat": 5,
+        "maxStat": 7
+      },
+      {
+        "stat": "% Air Resistance",
+        "minStat": 5,
+        "maxStat": 7
+      }
+    ],
+    "customStats": {},
+    "conditions": {
+      "conditions": {},
+      "customConditions": {}
+    },
+    "imageUrl": "item/1321.png"
+  }, {
+    "dofusID": "25753",
+    "name": {
+      "en": "Bontarian Rat Ring",
+      "fr": "Anneau du Rat Bontarien",
+      "de": "[!] Anneau du Rat Bontarien",
+      "es": "[!] Anneau du Rat Bontarien",
+      "it": "[!] Anneau du Rat Bontarien",
+      "pt": "[!] Anneau du Rat Bontarien"
+    },
+    "itemType": "Ring",
+    "setID": null,
+    "level": 110,
+    "stats": [
+      {
+        "stat": "Vitality",
+        "minStat": 51,
+        "maxStat": 100
+      },
+      {
+        "stat": "Chance",
+        "minStat": 21,
+        "maxStat": 30
+      },
+      {
+        "stat": "Wisdom",
+        "minStat": 11,
+        "maxStat": 20
+      },
+      {
+        "stat": "Power",
+        "minStat": 11,
+        "maxStat": 20
+      },
+      {
+        "stat": "Range",
+        "minStat": null,
+        "maxStat": 1
+      },
+      {
+        "stat": "Water Damage",
+        "minStat": 3,
+        "maxStat": 4
+      },
+      {
+        "stat": "Prospecting",
+        "minStat": 4,
+        "maxStat": 5
+      },
+      {
+        "stat": "Initiative",
+        "minStat": 151,
+        "maxStat": 200
+      },
+      {
+        "stat": "MP Reduction",
+        "minStat": 4,
+        "maxStat": 6
+      },
+      {
+        "stat": "AP Parry",
+        "minStat": 4,
+        "maxStat": 5
+      },
+      {
+        "stat": "Critical Resistance",
+        "minStat": -8,
```

Comment:
-11 is min, -8 is max

Reply:
I'm guessing this is because -8 is the one closest to 0, therefore we count it as a bigger stat?

Reply:
Correct